make_celery.py

```diff
@@ -1,4 +1,4 @@
-from src import create_app
+from src.communicator.communicator import celery
 
-flask_app = create_app()
-celery_app = flask_app.extensions["celery"]
+if __name__ == "__main__":
+    celery.start()
```

Comment:
Should I place it into its own file? That's probably a good idea. I put it there because it's part of the communicator's functionality to create new processes. Other files shouldn't have access to it.